lib/tapjoy/autoscaling_bootstrap/elb.rb

```diff
@@ -2,16 +2,30 @@ module Tapjoy
   module AutoscalingBootstrap
     # This class configures elastic load balancers
     class ELB
-      # Initialize the class
-      def initialize
+      def initialize(elb_hash, clobber_elb, zones, security_groups)
+        elb_config = build_config(elb_hash, zones, security_groups)
+        check_valid_config(elb_config)
+
+        if exists && clobber_elb
+          delete
+        elsif exists
+          return
```

Comment:
Don't quote me on this, but I feel like ```return``` is unnatural in initializers - The better approach might be to optionally choose to run create(elb_config), and let the method return naturally as it hits the last line.